source/Willow-Bootstrap/BootstrapHamburgerButton.class.st

```diff
@@ -35,16 +34,13 @@ BootstrapHamburgerButton >> initializeTargeting: aStringOrSymbol [
 { #category : #rendering }
 BootstrapHamburgerButton >> renderContentOn: aCanvas [
 
-	| button |
-
-	button := aCanvas button.
-	button bePush.
-	self buttonCommand applyTo: button on: aCanvas.
-	button with: [ self renderIconOn: aCanvas ]
+	aCanvas button
+		bePush;
+		with: [ self renderIconOn: aCanvas ] applying: self buttonCommand
```

Comment:
Since there is already a cascade of messages here, I would change it to ` bePush; apply: ... ; with: ...`